remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
Correct. I am saying that *this is expected and correct behavior* since the terminal is rewriting with `\r` and splitting lines *should not be done*.

This is a special case where a rewrite is being done on the same line. No correction is necessary there.